### PR TITLE
chore(deps): bump sanity libs from 2.29.8 to 2.30.0

### PR DESCRIPTION
**Pull request by artzag**
---
added 'upgrade:sanity' script upgrading minor sanity dependencies in root and all packages

## Checklist
I have:
- [x] used [conventional commit messages](https://www.conventionalcommits.org/en/v1.0.0/#summary)
- [x] committed with git-hooks active (ran npm install at least once locally before committing)
- [x] considered if this is a breaking change
- [x] tested the changes using npm link / yarn link
- [x] triggered nrkno-sanity-libs-releaser for this branch and checked the status

The branch-build must be manually triggered by the nrk.no team using nrkno-sanity-libs-releaser.

After merging, new version(s) should be published to npm by by triggering
nrkno-sanity-libs-releaser for the master branch.

These manual steps are a safeguard to prevent accidental releases and version changes.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -59,7 +60,8 @@
     "eslint:check": "eslint . --ext .js,.ts,.tsx --max-warnings=0",
     "prettier:check": "prettier --check .",
     "package:lib": "hygen package ts && npm run init",
-    "package:sanity-plugin": "hygen package sanity-plugin && npm run init"
+    "package:sanity-plugin": "hygen package sanity-plugin && npm run init",
+    "upgrade:sanity": "ncu /@sanity/ -u -t minor && lerna exec -- ncu /@sanity/ -u -t minor && npm install && npm run init"
```
Comment (rosvoll) [inline package.json:64]: oi, hva gjør denne?

Comment (artzag) [reply package.json:64]: Oppgraderer Sanity biblioteker i alle pakker ;) Har lagt til kort forklaring i README. Også - har brukt scripten for å oppdatere alt til 2.30.0 nå.

Comment (rosvoll) [reply package.json:64]: Nice!

**Review by rosvoll — APPROVED** [review]
Veldig bra! 👏 
(Men få noen annens godkjenning her også, siden det fort kan være noe jeg ikke kjenner til her)

**Review by pingrid — APPROVED** [review]
Ser veldig bra ut 👍

**Pull request merged**